ucsmsdk/utils/comparesyncmo.py

```diff
@@ -144,19 +144,19 @@ def _translate_managed_object(mo, xlate_org, xlate_map):
         diff_dn = mo.dn
         if diff_dn in xlate_map:
             ref_dn = xlate_map[diff_dn]
+            _update_mo_dn_and_naming_props(mo, ref_dn)
         else:
             diff_dn = re.sub(r'[/]*[^/]+$', '', diff_dn)
-            while diff_dn is not None or diff_dn == "":
+            while diff_dn:
                 if diff_dn not in xlate_map:
```

Comment:
@ragupta-git Isn't `if diff_dn not in xlate_map:` redundant here? The flow would come to else only when `diff_dn not in xlate_map`